lib/ocamlorg_web/graphql.mli

```diff
@@ -0,0 +1,9 @@
+(* This function compares two string and returns true if they are equal *)
+val starts_with : string -> string -> bool
+
+(* This function compares two string and returns true if they are equal *)
+val is_package : string -> string -> bool
+
+(* This schema allows to query for opam packages from the opam-repository and
+   returns graphql data as graphiql playground and grphql json data*)
```

Comment:
Is "and returns graphql data as graphiql playground and grphql json data" part of the schema definition? I was under the impression that that was done in `router.ml`.